plugins/hydra_rq_launcher/hydra_plugins/hydra_rq_launcher/rq_launcher.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import logging
+from typing import Any, Optional, Sequence
+
+from hydra.core.config_loader import ConfigLoader
+from hydra.core.utils import JobReturn
+from hydra.plugins.launcher import Launcher
+from hydra.types import TaskFunction
+from omegaconf import DictConfig
+
+log = logging.getLogger(__name__)
+
+
+class RQLauncher(Launcher):
+    def __init__(self, **kwargs: Any) -> None:
```

Comment:
I think params is a better name here.
```suggestion
    def __init__(self, **params: Any) -> None:
```